contentcuration/contentcuration/dev_settings.py

```diff
@@ -1,8 +1,23 @@
 # flake8: noqa
+from dotenv import load_dotenv
+
 from .not_production_settings import *  # noqa
 
 DEBUG = True
 
 ROOT_URLCONF = "contentcuration.dev_urls"
 
 INSTALLED_APPS += ("drf_yasg",)
+
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.IsAuthenticated',
+    ),
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework.authentication.SessionAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
+        'rest_framework.authentication.TokenAuthentication',
+    )
+}
+
+load_dotenv(override=True)  # take environment variables from .env.
```

Comment:
What is this for and the package you added? What's the reasoning?

Reply:
This package takes env vars from a `.env` file placed at the top level directory of project. It helps in setting/updating envniornment variables easily. The need arose when I wanted to switch between GCloud SQL and local database. 

Earlier, I was playing with my bash profile files. Chaging them constantly isn't a good practice. After this change, its a breeze to create, change or update any env vars I want to. This will help our new hires too! I can guarantee that.

Reply:
Okay, sounds good. Personally, I've used [direnv](https://direnv.net/) for contextualizing environment variables through an `.envrc` file. I have no problem with this though, but I think visibility into this capability is lacking in this PR in its current state. You should consider adding some documentation to the README with an example of how this could be useful for other devs.

Reply:
Honestly, I feel direnv is better because dotenv requires changing the project's files whereas direnv is completely agnostic about the project and language. I'll shift to direnv and will also add documentation to README about direnv.